deploy/docker/entrypoint.sh

```diff
@@ -2,6 +2,102 @@
 
 set -e
 
+init_env_file() {
+  CONF_PATH="/appsmith-stacks/configuration"
+  ENV_PATH="$CONF_PATH/docker.env"
+  TEMPLATES_PATH="/opt/appsmith/templates"
+  echo "Initialize .env file"
+  if ! [[ -e "$ENV_PATH" ]]; then
+    # Generate new docker.env file when initializing container for first time or in Heroku which does not have persistent volume
+    echo "Generating default configuration file"
+    mkdir -p "$CONF_PATH"
+    AUTO_GEN_MONGO_PASSWORD=$(
+      tr -dc A-Za-z0-9 </dev/urandom | head -c 13
+      echo ''
+    )
+    AUTO_GEN_ENCRYPTION_PASSWORD=$(
+      tr -dc A-Za-z0-9 </dev/urandom | head -c 13
+      echo ''
+    )
+    AUTO_GEN_ENCRYPTION_SALT=$(
+      tr -dc A-Za-z0-9 </dev/urandom | head -c 13
+      echo ''
+    )
+    bash "$TEMPLATES_PATH/docker.env.sh" > "$TEMPLATES_PATH/docker.env.temp"
+    
+    if [[ -z "${APPSMITH_DISABLE_TELEMETRY}" ]]; then
+      export APPSMITH_DISABLE_TELEMETRY=false
+    fi
+
+    if [[ -z "${APPSMITH_MONGODB_URI}" ]]; then
+      export APPSMITH_MONGODB_URI=mongodb://appsmith:$AUTO_GEN_MONGO_PASSWORD@localhost/appsmith
```

Comment:
The password might contain special characters. Wrapping in `"` won't fix the case where the password contains `"` itself, but it solves 90% of the cases, including `'` or the space character.

```suggestion
      export APPSMITH_MONGODB_URI="mongodb://appsmith:$AUTO_GEN_MONGO_PASSWORD@localhost/appsmith"
```